tools/ci_setup.ml

```diff
@@ -0,0 +1,296 @@
+index c6d09fb..61b1e5b 100644
+--- a/test/src/test_bignum.ml
++++ b/test/src/test_bignum.ml
+@@ -3,6 +3,11 @@ open! Expect_test_helpers_core
+ open Bignum
+ open Bignum.For_testing
+ 
++module Zarith = struct
++  module Q = Q
++  module Z = Z
++end
++
+ let%expect_test "Bignum.abs" =
+   let test t =
+     let t' = require_no_allocation [%here] (fun () -> abs t) in
+@@ -71,7 +76,7 @@ let%expect_test "Bignum.sexp_of_t does use Scientific Notation" =
+ let compare_floats ~of_float x =
+   let x' = x |> of_float |> Bignum.to_float in
+   if not (Float.( = ) x x' || (Float.is_nan x && Float.is_nan x'))
+-  then raise_s [%message "mismatch" (x : float) (x' : float)]
++  then raise_s [%message "mismatch" (x : float) (x' : float) (x |> of_float : Bignum.t)]
+ ;;
+ 
+ let%expect_test "roundtrip: f |> Bignum.of_float_decimal |> Bignum.to_float" =
+@@ -783,7 +788,7 @@ let%test_module _ =
+          -1073741825 -> ( 6) \001\253\255\255\255\191 |}]
+     ;;
+ 
+-    let%expect_test ("bin_io serialization V2 (javascript)" [@tags "js-only"]) =
++    let%expect_test ("bin_io serialization V2 (javascript)" [@tags "js-only", "no-wasm"]) =
+       bin_io_tests (module V2);
+       [%expect
+         {|
+@@ -811,6 +816,34 @@ let%test_module _ =
+          -1073741825 -> ( 6) \001\253\255\255\255\191 |}]
+     ;;
+ 
++    let%expect_test ("bin_io serialization V2 (Wasm)" [@tags "wasm-only"]) =
++      bin_io_tests (module V2);
++      [%expect
++        {|
++                   0 -> ( 1) \000
++                   1 -> ( 2) \001\001
++                  -1 -> ( 3) \001\255\255
++           100000001 -> ( 6) \001\253\001\225\245\005
++           1000000.1 -> ( 6) \002\253\129\150\152\000
++           100000.01 -> ( 6) \003\253\129\150\152\000
++           10000.001 -> ( 6) \004\253\129\150\152\000
++           1000.0001 -> ( 6) \005\253\129\150\152\000
++           100.00001 -> ( 6) \006\253\129\150\152\000
++           10.000001 -> ( 6) \007\253\129\150\152\000
++           1.0000001 -> ( 6) \008\253\129\150\152\000
++          0.10000001 -> ( 6) \009\253\129\150\152\000
++         0.010000001 -> (11) \010\253\129\150\152\000\253\000\202\154\059
++        0.0010000001 -> (22) \011\02010000001\04710000000000
++      10000000000000 -> (16) \011\01410000000000000
++     -10000000000000 -> (17) \011\015\04510000000000000
++12345678901234567.12345678901234567 -> (55) \01151234567890123456712345678901234567\047100000000000000000
++       1099511627775 -> (15) \011\0131099511627775
++          1073741823 -> ( 6) \001\253\255\255\255\063
++         -1073741824 -> ( 6) \001\253\000\000\000\192
++          1073741824 -> (12) \011\0101073741824
++         -1073741825 -> (13) \011\011\0451073741825 |}]
++    ;;
++
+     let%expect_test "bin_io de-serialization V2" =
+       (* Some bignums will have two bin_io representation depending on where their
+          were serialized.  Make sure we're able to parse things back regardless of the
+|bignum}
```

Comment:
https://github.com/orgs/community/discussions/42534

Reply:
https://discuss.ocaml.org/t/syntax-highlighting-on-github/11656/5

Reply:
I have removed this piece of comment.